k8s/backend-deployment.yml

```
@@ -0,0 +1,35 @@
+  namespace: chat-app
+
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: backend
+  template:
+    metadata:
+      name: backend-pod
+      namespace: chat-app
+      labels: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)